### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by tconlin**
## Describe your changes
- Add 404 Page and handle routing from Connections page and Integrations page
- Add padding to dashboard content
- Updated empty state working on connections page

## Issue ticket number and link

https://linear.app/nango/issue/NAN-570/design-404-page
https://linear.app/nango/issue/NAN-567/wording-on-empty-states

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/webapp/src/pages/Connection/Show.tsx [hunk 1]
```diff
@@ -60,8 +60,9 @@ export default function ShowIntegration() {
 
         const getConnections = async () => {
             const res = await getConnectionDetailsAPI(connectionId, providerConfigKey, false);
-
-            if (res?.status === 200) {
+            if (res?.status === 400) {
+                navigate('/404');
```
Comment (bodinsamuel) [inline packages/webapp/src/pages/Connection/Show.tsx]: you don't want to redirect to a 404, you want to keep the URL so the user can know that the ressource they accessed is 404. 
To achieve this, you can remove the route and instead store a state to do something like this in the component
```tsx
if (is404) {
  return <NotFound />
}
```

File: packages/webapp/src/pages/PageNotFound.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import DefaultLayout from '../layout/DefaultLayout';
+import { Link } from 'react-router-dom';
+
+export default function PageNotFound() {
+    return (
+        <>
```
Comment (bodinsamuel) [inline packages/webapp/src/pages/PageNotFound.tsx]: Unnecessary <>

**Review by bodinsamuel — COMMENTED** [review]
Thanks for handling that, just a small comments on the implem

File: .env-example
Comment (bodinsamuel) [inline .env-example:1]: Why did you rename this file?

File: packages/webapp/src/pages/Connection/Show.tsx (hunk 1 above)
Comment (tconlin) [reply packages/webapp/src/pages/Connection/Show.tsx]: Implemented this

File: .env-example
Comment (tconlin) [reply .env-example:1]: mistake, fixed

File: packages/webapp/src/App.tsx [hunk 3]
```diff
@@ -120,6 +121,7 @@ const App = () => {
                             <Route path="/reset-password/:token" element={<ResetPassword />} />
                         </>
                     )}
+                    <Route path="/404" element={<PageNotFound />} />
```
Comment (bodinsamuel) [inline packages/webapp/src/App.tsx]:
```suggestion

```

**Review by bodinsamuel — APPROVED** [review]
🚀

**Review by bodinsamuel — APPROVED** [review]

**Comment by bodinsamuel** [comment]
Sorry about the delay Taylor. Don't hesitate to ping on slack or ask the others too 👍🏻
